src/homework/java/jakir/Decision_Structure/PrintNumberInWord.java

```diff
@@ -0,0 +1,43 @@
+package homework.java.jakir.Decision_Structure;
+
+public class PrintNumberInWord {
+
+    public static void main(String[] args) {
+
+        byte num = 5;
+
+
+        if(num == 1) {
```

Comment:
This is what we are NOT looking for. For clarification please consult with the team.